src/emulate.c

```diff
@@ -81,6 +87,13 @@ static inline void update_time(riscv_t *rv)
     rv->csr_time[1] = t >> 32;
 }
 
+#if RV32_HAS(SYSTEM)
+static inline void get_time_now(struct timeval *tv)
```

Comment:
It is not worthy to have an inline function invoking a single internal function.